src/main/res/values/strings.xml

```diff
@@ -23,14 +23,14 @@
     <string name="compiled_for_debugging">Compiled for debugging</string>
     <string name="confirmation">Confirmation</string>
     <string name="connecting">Connecting…</string>
-    <string name="contains_non_free_media">Contains non-free media</string>
+    <string name="contains_non_free_media">Contains non-libre media</string>
     <string name="could_not_download_FORMAT">Could not download %s</string>
     <string name="could_not_sync_FORMAT">Could not sync %s</string>
     <string name="could_not_validate_FORMAT">Could not validate %s</string>
-    <string name="credits">Credits</string>
+    <string name="credits">Contributors</string>
     <string name="dark">Dark</string>
-    <string name="delete">Delete</string>
```

Comment:
I think we'll need to revert this and add a remove string, as this would be confusing as delete would be the correct word in some contexts.

Reply:
OK. I thought it was just for removing the repository, where "delete" only works if you think of it the right way.